docs/testing.md

```diff
@@ -1,5 +1,106 @@
 # Testing
 
+## Karma setup for Typescript
+
+Webpacker does not setup `Karma` by default, so you've to manually install it along with its dependencies as per your need. Following things marked as optional can be used to fancify the test results (Recommended).
+
+```js
+// package.json
+"scripts": {
+  "test": "NODE_ENV=test karma start"
+},
+"dependencies": {
+  "typescript": "^2.5.2",
+  "ts-loader": "^2.3.7",
+  "karma": "^1.7.1",
```

Comment:
`karma` and family should be under devDependencies, no?

Reply:
Yes. Done.